app/src/main/java/org/simple/clinic/summary/teleconsultation/contactdoctor/PhoneNumberListItem.kt

```diff
@@ -0,0 +1,38 @@
+package org.simple.clinic.summary.teleconsultation.contactdoctor
+
+import android.annotation.SuppressLint
+import androidx.recyclerview.widget.DiffUtil
+import io.reactivex.subjects.Subject
+import kotlinx.android.synthetic.main.list_contact_doctor_phone_number.*
+import org.simple.clinic.R
+import org.simple.clinic.summary.teleconsultation.api.TeleconsultPhoneNumber
+import org.simple.clinic.widgets.ItemAdapter
+import org.simple.clinic.widgets.recyclerview.ViewHolderX
+
+data class PhoneNumberListItem(
+    val phoneNumber: TeleconsultPhoneNumber
+) : ItemAdapter.Item<TeleconsultPhoneNumber> {
+
+  object DiffCallback : DiffUtil.ItemCallback<PhoneNumberListItem>() {
+    override fun areItemsTheSame(oldItem: PhoneNumberListItem, newItem: PhoneNumberListItem): Boolean {
+      return oldItem.phoneNumber == newItem.phoneNumber
+    }
+
+    override fun areContentsTheSame(oldItem: PhoneNumberListItem, newItem: PhoneNumberListItem): Boolean {
+      return oldItem == newItem
+    }
+  }
+
+  override fun layoutResId(): Int = R.layout.list_contact_doctor_phone_number
+
+  @SuppressLint("SetTextI18n")
+  override fun render(holder: ViewHolderX, subject: Subject<TeleconsultPhoneNumber>) {
+    holder.phoneNumberIndexTextView.text = "${getAlphabetIndex(holder.adapterPosition)}."
+    holder.phoneNumberTextView.text = phoneNumber.phoneNumber
+    holder.itemView.setOnClickListener { subject.onNext(phoneNumber) }
```

Comment:
Can we create a specific event type (`NumberClicked(String)`) instead of coupling the event type to a domain type? They represent different things and they should be different.

Reply:
Done